SECURITY.md

```diff
@@ -7,15 +7,16 @@ lighty version contains 3 numbers representing MAJOR.MINOR.PATCH (X.Y.Z) levels.
 MAJOR number is mapped to major OpenDaylight release
 | lighty | OpenDaylight    |
 |--------|-----------------|
+| 16.Y.Z | Sulfur (16)     |
 | 15.Y.Z | Phosphorus (15) |
 | 14.Y.Z | Silicon (14)    |
```

Comment:
Please remove `14.Y.Z | Silicon (14)`. We have support only for two last releases.

Reply:
Done